third_party/musl/arch/mips/bits/alltypes.h

```diff
@@ -12,10 +12,14 @@
 #define __LONG_MAX 0x7fffffffL
 
 #ifndef __cplusplus
+#if defined(USE_COBALT_CUSTOMIZATIONS)
```

Comment:
We don't use mips port, no need to modify this

Reply:
@y4vor  mentioned modifying all platforms for consistency go/lbreview/260401.
Another reason is a future platform may need it.

I don't have very strong opinions on this though. @y4vor  thoughts ?

Reply:
We don't use mips. By all platforms I means the one's we support.

Reply:
Ill remove this